superbench/benchmarks/directx_benchmarks/DirectXGPUCoreFLOPs/GPUCoreFLOPs.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
Do we need sln?

Reply:
yes, we need this for MSBUILD and also for development usage